pkg/errors/errors.go

```diff
@@ -73,6 +73,22 @@ func IsNotFound(err error) bool {
 	return err != nil && strings.Contains(err.Error(), "not found")
 }
 
+// IsTransient returns true if err represents a transient error
+func IsTransient(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	switch {
+	case strings.Contains(err.Error(), "operation time out"),
```

Comment:
these are the errors I've seen when the network is not ready, or offline (e.g. when changing networks)